kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/PaintConvertUtil.java

```diff
@@ -0,0 +1,145 @@
+package com.oracle.javafx.scenebuilder.kit.util;
+
+import javafx.scene.paint.*;
+
+import java.util.List;
+
+/**
+ * Convert Paint to java code or css code;
+ */
+public class PaintConvertUtil {
+
+    private static final int ROUNDING_FACTOR = 10000;//Use for round to 4 decimal places
+
+    public static String convertPaintToCss(Paint fxPaint) {
+        if (fxPaint == null) {
+            return "";
+        }
+        if (fxPaint instanceof LinearGradient) {
+            LinearGradient paint = (LinearGradient) fxPaint;
+            StringBuilder strBuilder = new StringBuilder("linear-gradient(from ")
+                    .append(lenToStr(paint.getStartX(), paint.isProportional()))
+                    .append(" ").append(lenToStr(paint.getStartY(), paint.isProportional()))
+                    .append(" to ").append(lenToStr(paint.getEndX(), paint.isProportional()))
+                    .append(" ").append(lenToStr(paint.getEndY(), paint.isProportional()))
+                    .append(", ");
+            connectCycleMethodAndStops(strBuilder, paint.getCycleMethod(), paint.getStops());
+            return strBuilder.toString();
+        } else if (fxPaint instanceof RadialGradient) {
```

Comment:
Is it possible to create a Paint object which is an instance of either `LinearGradient` or `RadialGradient` from the Scene Builder's ColorPicker dialog?

Reply:
There is a PaintPickerController object in ColorPicker, we can get the current Paint (Color or LinearGradient or RadialGradient) of ColorPicker through PaintPickerController